src/github.com/matrix-org/dendrite/common/config/appservice.go

```diff
@@ -51,6 +59,10 @@ type ApplicationService struct {
 	// Information about an application service's namespaces. Key is either
 	// "users", "aliases" or "rooms"
 	NamespaceMap map[string][]ApplicationServiceNamespace `yaml:"namespaces"`
+	// Whether rate limiting is applied to each application service user
+	RateLimited bool `yaml:"rate_limited"`
```

Comment:
I think this should default to true?

Reply:
In true go fashion this defaults to false. Not sure if we can check whether it's false because it hasn't been set, or because the user set it to false.